.github/workflows/translation-advice.yml

```diff
@@ -0,0 +1,37 @@
+name: authentik-translation-advice
+
+on:
+  pull_request:
+    branches:
+      - main
+      - translations-comments
+    paths:
+      - "/locale/**"
+      - "/web/src/locales/**"
+
+jobs:
+  post-comment:
+    runs-on: ubuntu-latest
+    steps:
+      - name: Find Comment
+        uses: peter-evans/find-comment@v2
+        id: fc
+        with:
+          issue-number: ${{ github.event.pull_request.number }}
+          comment-author: "github-actions[bot]"
+          body-includes: authentik translations instructions
+      - name: Create or update comment
+        uses: peter-evans/create-or-update-comment@v2
+        with:
+          comment-id: ${{ steps.fc.outputs.comment-id }}
+          issue-number: ${{ github.event.pull_request.number }}
+          edit-mode: replace
+          body: |
+            authentik translations instructions
+
+            Thanks for your pull request!
+
+            Authentik translations are handled using
```

Comment:
Very small nitpick; authentik should be stylised lowercase here

Reply:
Fixed